trustpinning/trustpin.go

```diff
@@ -47,6 +49,7 @@ func NewTrustPinChecker(trustPinConfig TrustPinConfig, gun string) (CertChecker,
 		caRootPool := x509.NewCertPool()
 		for _, caCert := range caCerts {
 			if err = trustmanager.ValidateCertificate(caCert); err != nil {
+				logrus.Debugf("Trust-pinning validation for child cert failed: %s", err)
```

Comment:
Many apologies - I think the above comment may be out of date.  This block only goes over all the CA certs listed in the root CA bundle and ensures that it's not expired and that it has a big enough RSA key, before using it to do any kind of validation of the root certs.

So it may make more sense to either leave out the message entirely or to say `"Ignoring root CA certificate ??? in bundle: %s", err"`  By ??? I mean some way of identifying the cert in the cert bundle - maybe it's fingerprint, or maybe something like 1 of 3, so you can identify which one in the bundle is invalid?